base/libdl.jl

```diff
@@ -205,6 +205,43 @@ if is_linux()
     end
 end # linux-only
 
+if Sys.KERNEL == :FreeBSD
```

Comment:
You don't need `@static` at the top level 99% of the time since that's already what the branch is doing.